vms/platformvm/add_subnet_validator_tx.go

```diff
@@ -193,7 +197,8 @@ func (tx *UnsignedAddSubnetValidatorTx) Execute(
 			)
 		}
 
-		baseTxCredsLen := len(stx.Creds) - 1
+		// Last credentials are node's credentials
```

Comment:
I wouldn't do such specialized things here, from concept sigs are always sorted.
Beside this we wanted to handle the node-sig as a normal signature, it can be a normal wallet, means that a node sig can also be a baseSig